src/chef/cfg/cfg.cpp

```diff
@@ -178,6 +178,41 @@ namespace chef {
 			});
 	}
 
+	std::set<cfg_token> first_plus_set(const cfg_var& var, const cfg_seq& rule,
+		const std::map<cfg_var, std::set<cfg_token>>& first_sets,
+		const std::map<cfg_var, std::set<cfg_token>>& follow_sets)
+	{
+		std::set<cfg_token> result;
+
+		bool all_erasable = false;
+		for (const auto& item : rule) {
+			const bool erasable = std::visit(detail::overload{
```

Comment:
Already has a using decl.

```suggestion
			const bool erasable = std::visit(overload{
```